js/scripts.js

```diff
@@ -1243,6 +1240,13 @@ $(document).ready(function() {
     if (isMobileDevice()) {
         $('#banner-bottom').removeClass('hide');
     }
+
+    // Initialize Masonry
+    if (window.imagesLoaded != undefined) {
```

Comment:
I think the more standard way of checking undefined objects in javascript is like this `if (typeof window.imagesLoaded !== 'undefined')`. this is common specially when js is set to strict mode.